sdk/python/tests/integration/e2e/test_universal_e2e.py

```diff
@@ -16,6 +16,7 @@
 @pytest.mark.parametrize("infer_features", [True, False])
 def test_e2e_consistency(environment, e2e_data_sources, infer_features):
     fs = environment.feature_store
+    fs.config.project = fs.config.project + str(infer_features)
```

Comment:
different bins for different tests
otherwise the old data remains in the integration online store